asc2qr.sh

```diff
@@ -45,38 +45,27 @@ if [ ! -f ${asc_key} ]; then
 	exit 1
 fi
 
-# Create a temp file to use as a pattern for splitting the input key file.
-# This helps protect against file collisions in the current directory.
-export TMPDIR=""
-tmp_file=$(mktemp keyparts.XXXXXX)
-if [ $? -ne 0 ]; then
-	echo "failed to create temporary file"
-	exit 1
-fi
-
-# Split the key file into usable chunks that the QR encoder can consume
-split -b ${file_split_size} ${asc_key} "${tmp_file}."
+## Split the key file into usable chunks that the QR encoder can consume
+chunks=()
+while true; do
+    s=$( head -c ${max_qr_bytes} )
```

Comment:
The 'head' call here doesn't correctly move through the file possibly because it interacts strangely with the 'cat' input of the key file. If you make the 'max_qr_bytes' value small to force chunking (I used 700 on a 4096-bit key file) the echo of $s is not correct -- the first part of the second chunk missing.